src/libraries/Microsoft.NETCore.Platforms/src/runtimeGroups.props

```diff
@@ -162,7 +162,7 @@
     <RuntimeGroup Include="freebsd">
       <Parent>unix</Parent>
       <Architectures>x64</Architectures>
-      <Versions>11;12;13</Versions>
+      <Versions>12;13</Versions>
```

Comment:
AFAIK we never remove old versions. It may not matter since we never shipped.

Reply:
Yes, we still don't have an SDK out in the wild for FreeBSD so I thought cleaning out as much old stuff will make future work easier - less technical debt too.
Let me know if I have to revert it though.